.agents/skills/superdesign/SUPERDESIGN.md

```
@@ -0,0 +1,502 @@
+```json
+[
+  {"name": "activeItem", "type": "string", "defaultValue": "home"},
+  {"name": "homeHref", "type": "string", "defaultValue": "#"},
+  {"name": "exploreHref", "type": "string", "defaultValue": "#"}
+]
+```
+
+---
+
+<marketing_assets_dimension_guidelines>
+| Category  | Platform               | Asset Type            | Aspect Ratio | Recommended Size (px) |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)